templates/js-crawlee-cheerio/src/main.js

```diff
@@ -7,19 +7,27 @@
 // For more information, see https://docs.apify.com/sdk/js
 import { Actor } from 'apify';
 // For more information, see https://crawlee.dev
-import { CheerioCrawler } from 'crawlee';
-import { router } from './routes.js';
+import { CheerioCrawler, Dataset } from 'crawlee';
 
 // Initialize the Apify SDK
 await Actor.init();
 
-const startUrls = ['https://apify.com'];
+const { startUrls =  ["https://crawlee.com"], maxRequestsPerCrawl = 100 } = await Actor.getInput();
```

Comment:
We don't need the defaults here, the input schema has it and locally we'll also have the `INPUT.json` prefilled with defaults. Let's just take the `startUrls` and `maxRequestsPerCrawl` from the input and use it.